app/_kong_plugins/loggly/examples/send-data-to-loggly.yaml

```
@@ -13,7 +13,7 @@ variables:
     value: $LOGGLY_SERVICE_TOKEN
 
 config:
-  key: ${key}
+  key: $KEY
```

```suggestion
  key: ${key}
```
this is pulling in a variable from `variables`, so it must be in this format.